tox.ini

```diff
@@ -12,12 +12,14 @@ setenv =
 deps =
 	-r{toxinidir}/requirements.txt
 	django111: Django<1.12
-	django20: Django<2.1
-	django21: Django<2.2
-	django-oauth-toolkit>=1.1.2,<1.2
-	djangorestframework>=3.7,<3.8
-	pyjwt>=1.6.4,<1.7
+	django22: Django<3
+	django30: Django>=3
```

Comment:
I suggest here `Django<3.1`

Reply:
seems a bit strange, since tox in docker can resolve it just fine without it...
But I will give it a try

Reply:
well wasn't that either,
but `django-oauth-tollkit` dropped `Django 1.11` on a previous release.

Reply:
would be a really goo time to drop it also...

Reply:
Well Python 3 is fine in travis-ci

But all hell breaks lose on Python 2.7